.github/workflows/pull-request.yml

```diff
@@ -47,6 +47,22 @@ jobs:
         run: |
           npm ci
           docker-compose build app
+      - name: Get installed Playwright version
+        id: playwright-version
+        # https://github.com/microsoft/playwright/issues/7249#issuecomment-1201476980
+        run: echo -n "::set-output name=version::$(npm ls @playwright/test --json | jq --raw-output '.dependencies["@playwright/test"].version')"
+
+        # https://github.com/microsoft/playwright/issues/7249#issuecomment-1154603556
+      - name: Cache playwright browsers
```

Comment:
Unfortunately, this does not seem to work for me (forked this repo and pushed twice; the browsers were also downloaded the second time)
![image](https://user-images.githubusercontent.com/43119728/191495729-00386844-87c5-43fa-9d41-5ff2b93f2634.png)

Reply:
@JeanneSon can you link me to the run? The cache only gets written to if the whole job was successful. Perhaps it failed?

Reply:
Nevermind, it's obviously pretty easy to find the run myself 😆.
Yeah, [the first execution failed](https://github.com/JeanneSon/fork-web-languageforge/actions/runs/3096685403/jobs/5012515627), so you can see that the "Post Cache playwright browsers" step didn't happen. That's when things get cached. The "Cache playwright browsers" step is when the cache gets loaded if it exists.
![image](https://user-images.githubusercontent.com/12587509/191688450-5b692d17-0260-43ed-bc80-659efcaa008b.png)

The more recent run is green, so if you trigger the action again it **will** use the cache. Pinky promise 😰.